chef/cookbooks/keystone/recipes/server.rb

```diff
@@ -545,6 +544,18 @@
     tenant_name tenant
     action :add_tenant
   end
+
+  ruby_block "saving id for default #{tenant} tenant" do
+    block do
+      # note that we know because of what's at the end of this recipe that the
+      # .openrc has been created, and that the openstack cli is available
+      tenant_id = %x[source /root/.openrc && openstack project show openstack | awk '/id/  { print $4 }'].chomp
```

Comment:
FWIW, I'm mixed on running this on every run: we're losing at least 3s because of this new code on each run. This is not really good.

The alternative is too only do this as a notification to the `keystone_register` above, but this wouldn't catch ID changes (in case the user deletes / recreates the tenant).